.github/workflows/integration_test.yml

```diff
@@ -45,4 +45,5 @@ jobs:
 
       - name: push to quay
         if: github.event_name == 'push'
-        run: podman push quay.io/sustainable_computing_io/kepler:latest
\ No newline at end of file
+        run: docker tag localhost:5001/kepler:devel quay.io/sustainable_computing_io/kepler:latest
```

Comment:
the image needs to be built first
https://github.com/sustainable-computing-io/kepler/pull/227

Reply:
I suppose we tested at line https://github.com/sustainable-computing-io/kepler/actions/runs/3098392762/jobs/5016249006#step:8:189

Reply:
#277 just test with image build but not with kind.

Reply:
the kind test works for x86, the Makefile build is to support multi arch build.